content/10-managing-databases/04-database-replication/01-database-replication-introduction.mdx

```diff
@@ -0,0 +1,76 @@
+
+* improve the availability of data
+* increase the speed of data access
+* enhance server performance
+* accomplish preparation for disaster recovery
+* improve analytics
+
+### Improve the availability of data
+
+*Replication* improves the availability of data because it ensures against relying on a single point being failing and losing database access. If there were some sort of technical glitch, power outage, or any number of different occurrences that affected that single point, then your application would not be able to access the data needed for its proper function. 
+
+Data replication enhances the resiliency and reliability of systens by storing data across multiple nodes. If one node goes down, there is another one to step up with the exact same data on it to keep operations running smoothly.
+
+### Increase the speed of data access
+
+For companies both large and small, there may be employees located globally. Database replication helps ensure that for any given employee in the company that needs particular data that they can get it timely and without too much delay.
+
+Replicating data to local servers in the locations near to where it is being accessed, provides users with faster data access and queries. If a company had a single node in the USA, then an employee trying to pull data in Europe could experience latency when trying to access it. A company with a cluster in the USA as well as a replicated cluster in Europe would ensure that this employee can work efficiently with the data that they need when they need it.
+
+### Enhance server performance
+
+Database replication also reduces the load on the primary server of the database. If data was only able to be accessed on a single server, that server would be handling every request coming to the database. With replication, that load can be dispersed among other nodes in the distributed system. Because all of the nodes have identical replicas of the data, some queries could get handled by secondary nodes and leave the primary to handle only certain reads or only writes.
+
+Saving the primary for handling write operations can prove to be beneficial because if it were handling all reads and writes that may put immense strain on the processing power. This could make sense for a situation where there are not many operations coming in to the database. However, in times of scaling and growth, it is best practice to delegate tasks across a distributed system rather than putting all the load on a single node's plate.
+
+### Accomplish preparation for disaster recovery
+
+In the case of catastrophe, replication also secures recovery of what gets lost or corrupted. It facilitates recovery by maintaining accurate backups at well-monitored locations. If one node in the distribution goes down for any reason, there are nodes at the ready to help recover what was lost because of their identical replicas.
```

Comment:
I think you need to be very careful in how you approach this one if only because a replica in itself is not considered a backup.  In fact, the very fact that they continuously apply changes can mean they're susceptible to recreating any bad state that the primary server passes to them.  I think there's two parts of this that are very relevant though and that you can highlight.

Replicas are typically used as targets for backups.  So for instance, when you want to back up your data, you turn off replication briefly on one of your followers to get the data in a consistent state, back up the data when there are no more write operations incoming, and then restart replication to help it catch back up on new updates.  The replica was able to give you a snapshot of consistent data to target without interrupting operations (assuming there are other read replicas or that the master can be briefly used for reads during this interim time).

The other way that replication can aid in disaster recovery is if the replica is specifically configured as a lagging follower.  You can configure certain followers to delay applying operations from the replication log by a certain amount of time (15 minutes, an hour or two, etc.).  That way, for instance, if the primary suffers some corruption event that it immediately replicates to its real-time followers, if the team notices the problem before the delayed replica is configured to apply those same changes, they can turn off the replication and have a mostly up-to-date server ready and running that hasn't applied the corrupting events.